README.md

```diff
@@ -42,6 +42,27 @@
 
 <br />
 
+## 🖱️ One command Boyka project setup
+
+First you need to install [Boyka command line assistant](https://github.com/BoykaFramework/boyka-cli) by running the following command:
```

Comment:
Install [Boyka command line assistant](https://github.com/BoykaFramework/boyka-cli) by running the following command: